recipes/r-sparlyr.nested/meta.yaml

```
@@ -0,0 +1,85 @@
+{% set version = '0.0.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sparklyr.nested
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/sparklyr.nested_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/sparklyr.nested/sparklyr.nested_{{ version }}.tar.gz
+  sha256: ecece96001c8c2acbe9dae2b68ac2dd166be038d2e92f06ce6cfa19200c1aca0
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-dplyr
+    - r-jsonlite
+    - r-listviewer
+    - r-purrr
+    - r-rlang
+    - r-sparklyr
+    - r-tidyselect
+  run:
+    - r-base
+    - r-dplyr
+    - r-jsonlite
+    - r-listviewer
+    - r-purrr
+    - r-rlang
+    - r-sparklyr
+    - r-tidyselect
+
+test:
+  commands:
+    - $R -e "library('sparklyr.nested')"           # [not win]
+    - "\"%R%\" -e \"library('sparklyr.nested')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=sparklyr.nested
+  license: Apache-2.0 | file LICENSE
```

The license file is a straight Apache-2.0 copy, so this can just be:

```suggestion
  license: Apache-2.0
```